docs/file/lib/tupelo.js.html

```diff
@@ -340,10 +347,14 @@
    */
 
   /**
-   * Resolve data from the user portion of the chain tree
+   * Resolve data from the user portion of the chain tree.
```

Comment:
maybe This should be:
`
Resolve data from the user portion ("/tree/data") of the chain tree.
`

Reply:
@tobowers Makes sense.